lib/resque_cleaner/server.rb

```diff
@@ -120,7 +120,7 @@ def text_filter(id, name, value)
           @stats = { :klass => {}, :exception => {} }
           @total = Hash.new(0)
           @jobs.each do |job|
-            klass = job["payload"]["class"] || 'UNKNOWN'
+            klass = job["payload"].to_h["class"] || 'UNKNOWN'
```

Comment:
Can you change this to `job.fetch('payload', {})['class'] || 'UNKNOWN'`? `to_h` looks bit too unclear on its purpose. Thanks!